manifests/02-ClusterRoleBinding.yaml

```diff
@@ -0,0 +1,12 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: rbac-permissions-operator
+subjects:
+- kind: ServiceAccount
+  name: rbac-permissions-operator
+  namespace: openshift-rbac-permissions
+roleRef:
+  kind: ClusterRole
+  name: rbac-permissions-operator
+  apiGroup: rbac.authorization.k8s.io
```

Comment:
please add `---
# operator can create rolebindings to admin clusterrole
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  name: rbac-permissions-operator-admin
subjects:
- kind: ServiceAccount
  name: rbac-permissions-operator
  namespace: openshift-rbac-permissions
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: admin`